packages/react/src/components/Icon/index.stories.js

```diff
@@ -2,25 +2,60 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import { select, withKnobs } from "@storybook/addon-knobs";
 
-import { ICONS, ICON_SIZES } from "../../constants";
+import Helper from "../Helper";
+import Input from "../Input";
+import Label from "../Label";
 import Icon from ".";
 
+import { ICONS, ICON_SIZES } from "../../constants";
+
 const icons = Object.keys(ICONS);
 const sizes = Object.values(ICON_SIZES);
 
-const stories = storiesOf("Icon", module);
+const IconsFilterer = () => {
+  const [filter, setFilter] = React.useState("");
 
-stories.addDecorator(withKnobs);
+  const displayedIcons = icons.filter(icon =>
+    icon.toLowerCase().includes(filter.toLowerCase())
+  );
+
+  const handleInputChange = e => setFilter(e.target.value);
+
+  return (
+    <div>
+      <Label htmlFor="icon-search">Search</Label>
+      <Input id="icon-search" onChange={handleInputChange} />
 
-stories.add("All", () => (
-  <div style={{ padding: "var(--space-xl)" }}>
-    {icons.map(icon => (
-      <div style={{ padding: "var(--space-s)", display: "inline-block" }}>
-        <Icon key={`Icon--${icon}`} icon={icon} />
+      <div
+        style={{
+          display: "flex",
+          flexWrap: "wrap",
+          padding: "var(--space-xl)",
+        }}
+      >
+        {displayedIcons.map(icon => (
+          <div
+            key={icon}
+            style={{
+              padding: "var(--space-s)",
+              width: 150,
+              textAlign: "center",
+            }}
+          >
+            <Icon key={`Icon--${icon}`} icon={icon} />
+            <Helper>{icon}</Helper>
+          </div>
+        ))}
       </div>
-    ))}
-  </div>
-));
+    </div>
+  );
+};
+
+const stories = storiesOf("Icons", module);
+
+stories.addDecorator(withKnobs);
```

Comment:
Hmm, so since we decided to add this decorator on global level. But apparently this change is in Checkbox component PR that has not yet been merged. Do you maybe wanna cherry pick this https://github.com/heetch/flamingo/pull/43/files#diff-2e6866a078d3aa8cca27fed58050c6cd and then remove this declaration in L56?
This PR won't need to get QAed so it will get merged faster

Reply:
That's a too big comment for me to cherry-pick. (That's why we should be careful with our commit history 🙊)
I'll just remove the `withKnobs` call here